docs/services/awssqs.md

```diff
@@ -0,0 +1,106 @@
+# AWS SQS 
+
+## Parameters
+
+This notification service is capable of sending simple messages to AWS SQS queue. 
+
+* `queue` - name of the queue you are intending to send messages to. Can be overwriten with target destination annotation.
+* `region` - region of the sqs queue can be provided via env variable AWS_DEFAULT_REGION
+* `key` - aws access key must be either referenced from a secret via variable or via env variable AWS_ACCESS_KEY_ID
```

Comment:
the key and secret are also optional?

Reply:
Yes, I would agree It could be considered optional if its otherwise provided, for example via environmental variables. Good point.